backend/api/PageApi/index.js

```diff
@@ -47,6 +55,18 @@ router.get('/courses/:id/review', authenticate, catchAsync(async (request, respo
   response.status(200).json({ courseUserIsLearning, problems });
 }));
 
+router.get('/courses/:id/review/persistent', authenticate, catchAsync(async (request, response) => {
+  const courseId = request.params['id'];
+
+  if (!(await canAccessCourse(courseId, request.currentUser))) {
+    return response.error(cantAccessError);
+  }
+
+  const courseUserIsLearning = await CourseUserIsLearningModel.select.oneByCourseIdAndUserId(courseId, request.currentUser.id);
+  const problems = await getLearnedProblemsByCuilId(courseId);
```

Comment:
`courseId` should be `courseUserIsLearning.id` - fixed in my branch, will push soon xx